# HG changeset patch
# User Mike Miller <mtmiller@octave.org>
# Date 1466537957 25200
#      Tue Jun 21 12:39:17 2016 -0700
# Node ID cfead9fdb9684c0104e333c646287ed4b6a0cf95
# Parent  ab2daf3ffcdad29d25f789394c15cdfecaa41b2f
Add more source files to the distribution

* Makefile.am (EXTRA_DIST): Include OctJavaQry.java and changelog.tmpl in the
list.
* etc/module.mk (etc_EXTRA_DIST): Include octave_branding_samples.svg in the
list.
* liboctave/cruft/amos/module.mk (liboctave_EXTRA_DIST): Include README in the
list.
* liboctave/cruft/ordered-qz/module.mk (liboctave_EXTRA_DIST): Include README
in the list.
* scripts/module.mk (scripts_EXTRA_DIST): Include scripts/@ftp/module.mk in the
list.
* test/module.mk (TEST_FILES): Include bug-38565.tst and command.tst in the
list.
* test/nest/module.mk (nest_TEST_FILES): Fix variable name typo.

diff --git a/Makefile.am b/Makefile.am
--- a/Makefile.am
+++ b/Makefile.am
@@ -42,24 +42,26 @@ BUILT_DISTFILES += \
 
 EXTRA_DIST += \
   CITATION \
   COPYING \
   INSTALL \
   NEWS \
   README \
   build-aux/OctJavaQry.class \
+  build-aux/OctJavaQry.java \
+  build-aux/changelog.tmpl \
   build-aux/check-subst-vars.in.sh \
   build-aux/find-files-with-tests.sh \
   build-aux/mk-default-qt-settings.in.sh \
   build-aux/mk-f77-def.in.sh \
   build-aux/mk-mxarray-h.in.sh \
-  build-aux/mk-version-h.in.sh \
   build-aux/mk-octave-config-h.sh \
   build-aux/mk-opts.pl \
+  build-aux/mk-version-h.in.sh \
   build-aux/move-if-change \
   build-aux/stl_algo.h-fixed \
   build-aux/subst-config-vals.in.sh \
   build-aux/subst-cross-config-vals.in.sh \
   build-aux/subst-default-vals.in.sh \
   build-aux/subst-f77-isnan-macro.in.sh \
   build-aux/subst-script-vals.in.sh \
   run-octave.in \
diff --git a/etc/module.mk b/etc/module.mk
--- a/etc/module.mk
+++ b/etc/module.mk
@@ -50,17 +50,18 @@ WINDOWS_PNG_ICONS = $(filter %-16.png %-
 BUILT_ICONS = \
   $(BUILT_PNG_ICONS) \
   etc/icons/octave-logo.ico
 
 etc_EXTRA_DIST += \
   $(BUILT_ICONS) \
   $(icon_IMAGE_FILES) \
   etc/icons/octave.appdata.xml.in \
-  etc/icons/octave.desktop.in
+  etc/icons/octave.desktop.in \
+  etc/icons/octave_branding_samples.svg
 
 image_DATA += \
   $(icon_IMAGE_FILES) \
   etc/icons/octave-logo.ico
 
 VENDOR = www.octave.org
 
 DIRSTAMP_FILES += \
diff --git a/liboctave/cruft/amos/module.mk b/liboctave/cruft/amos/module.mk
--- a/liboctave/cruft/amos/module.mk
+++ b/liboctave/cruft/amos/module.mk
@@ -60,8 +60,11 @@ CRUFT_SOURCES += \
   liboctave/cruft/amos/zunhj.f \
   liboctave/cruft/amos/zuni1.f \
   liboctave/cruft/amos/zuni2.f \
   liboctave/cruft/amos/zunik.f \
   liboctave/cruft/amos/zunk1.f \
   liboctave/cruft/amos/zunk2.f \
   liboctave/cruft/amos/zuoik.f \
   liboctave/cruft/amos/zwrsk.f
+
+liboctave_EXTRA_DIST += \
+  liboctave/cruft/amos/README
diff --git a/liboctave/cruft/ordered-qz/module.mk b/liboctave/cruft/ordered-qz/module.mk
--- a/liboctave/cruft/ordered-qz/module.mk
+++ b/liboctave/cruft/ordered-qz/module.mk
@@ -1,5 +1,8 @@
 CRUFT_SOURCES += \
   liboctave/cruft/ordered-qz/dsubsp.f \
   liboctave/cruft/ordered-qz/exchqz.f \
   liboctave/cruft/ordered-qz/ssubsp.f \
   liboctave/cruft/ordered-qz/sexchqz.f
+
+liboctave_EXTRA_DIST += \
+  liboctave/cruft/ordered-qz/README
diff --git a/scripts/module.mk b/scripts/module.mk
--- a/scripts/module.mk
+++ b/scripts/module.mk
@@ -39,16 +39,17 @@ include scripts/statistics/tests/module.
 include scripts/strings/module.mk
 include scripts/testfun/module.mk
 include scripts/time/module.mk
 
 ## include scripts/@ftp/module.mk
 ## The include above fails because Automake cannot process the '@' character.
 ## As a work around, the contents of scripts/@ftp/module.mk are placed directly
 ## in this module.mk file.
+scripts_EXTRA_DIST += scripts/@ftp/module.mk
 ######################## include scripts/@ftp/module.mk ########################
 FCN_FILE_DIRS += scripts/@ftp
 
 scripts_@ftp_FCN_FILES = \
   scripts/@ftp/ascii.m \
   scripts/@ftp/binary.m  \
   scripts/@ftp/cd.m  \
   scripts/@ftp/close.m  \
diff --git a/test/module.mk b/test/module.mk
--- a/test/module.mk
+++ b/test/module.mk
@@ -3,18 +3,20 @@ test_EXTRA_DIST =
 test_CLEANFILES =
 test_DISTCLEANFILES =
 test_MAINTAINERCLEANFILES =
 
 TEST_FILES += \
   test/fntests.m \
   test/args.tst \
   test/bug-31371.tst \
+  test/bug-38565.tst \
   test/bug-38576.tst \
   test/colormaps.tst \
+  test/command.tst \
   test/complex.tst \
   test/diag-perm.tst \
   test/error.tst \
   test/eval-catch.tst \
   test/for.tst \
   test/func.tst \
   test/global.tst \
   test/if.tst \
diff --git a/test/nest/module.mk b/test/nest/module.mk
--- a/test/nest/module.mk
+++ b/test/nest/module.mk
@@ -1,9 +1,9 @@
-nest_fcn_files = \
+nest_TEST_FILES = \
   test/nest/arg_nest.m \
   test/nest/arg_ret.m \
   test/nest/nest_eval.m \
   test/nest/no_closure.m \
   test/nest/persistent_nest.m \
   test/nest/recursive_nest.m \
   test/nest/recursive_nest2.m \
   test/nest/recursive_nest3.m \
